ERCS/erc-9999.md

```diff
@@ -0,0 +1,648 @@
+Title: Router Proxy
```

Comment:
```suggestion
---
eip: 7771
Title: Router Proxy
```
Assigning next sequential EIP/ERC/RIP number.

Please also update the filename.